infrastructure/async/src/main/java/tech/pegasys/teku/infrastructure/async/MetricTrackingExecutorFactory.java

```diff
@@ -66,11 +66,14 @@ public ExecutorService newCachedThreadPool(
             threadFactory);
     executor.allowCoreThreadTimeOut(true);
 
-    metricsSystem.createIntegerGauge(
-        TekuMetricCategory.EXECUTOR,
-        name + "_queue_size",
-        "Current size of the executor task queue",
-        () -> executor.getQueue().size());
+    metricsSystem
+        .createLabelledGauge(
+            TekuMetricCategory.EXECUTOR,
+            "queues",
+            "Current queues size for this executor",
+            "queue_size")
+        .labels(() -> executor.getQueue().size(), "queue_size");
```

Comment:
`queues` probably isn't the right metric name for this.  I'd suggest `executor_queue_size`.  The label name would be `executor` and the label (passed into the `.labels(...)` call would be `name`.  Currently this will break startup because we're trying to define multiple labels all with the same value of `"queue_size"`.